packages/suite/src/components/suite/StakingProcess/InfoRow.tsx

```diff
@@ -0,0 +1,76 @@
+import { Badge, Column, Paragraph, Row, useElevation } from '@trezor/components';
+import { Elevation, mapElevationToBorder, spacings, spacingsPx } from '@trezor/theme';
+import styled from 'styled-components';
+import { Subheading } from './Subheading';
+import { RowContent, RowSubheading } from './types';
+
+const ListItem = styled.li<{
```

Comment:
This component is not needed, just use `List.Item`

Reply:
removed, https://github.com/trezor/trezor-suite/pull/14713/commits/aa21e6e49b2370a3e9bbb5035db474d51a1e8519